component-framework/resources/GridCustomizerControlTemplate/PAGridCustomizer/types.ts

```diff
@@ -198,4 +198,5 @@ export type ColumnDataType =
   | 'DateAndTime'
   | 'Image'
   | 'File'
-  | 'Persona';
+  | 'Persona'
+  | 'RichText';
```

Comment:
thats right. "RichText" and "UniqueIdentifier" are two new types recently added. 
you can also add "UniqueIdentifier" to these types if you like.